src/lib/rmt/maria_db/current_root_password_dialog.rb

```diff
@@ -28,7 +28,7 @@ def initialize
     super
 
     @dialog_heading = N_('Database root password is required')
-    @dialog_label = N_('Please provide the current database root password.')
+    @dialog_label = N_('In order to setup RMT, database access is required. Please provide the current database root password.')
```

Comment:
I'm not sure this is necessary. The bug reporter didn't understand the difference between root password dialog and RMT user dialog -- doesn't mean that we have to change anyhing.
(`In order to setup RMT, database access is required.` -- this should already be pretty obvious. Sure, this is why you are running this Yast module in the first place :-))

Reply:
I know, I tought make everybody happy and explain this a little further. I let it like this, since it does not harm anybody